### PR TITLE
Defer javascript loading feature

### PR DESCRIPTION
**Pull request by cchabilall83**
This gives a site admin the ability to toggle javascript loading on/off.
- requires follow up issue for Todd to run a jobber
- ~pending article https://success.vanillaforums.com/kb/articles/add?draftID=606~
- ~translations https://github.com/vanilla/locales/pull/242~

Closes https://github.com/vanilla/vanilla/issues/9366

### CODE REVIEW

File: applications/dashboard/controllers/class.settingscontroller.php [hunk 1]
```diff
@@ -558,6 +558,18 @@ public function branding() {
...
                     'AllowEmpty' => true,
                 ]
+            ],
+            'Feature.DeferredLegacyScripts.Enabled' => [
+                'LabelCode' => t('Defer Javascript Loading'),
+                'Control' => 'toggle',
+                'Default' => true,
+                'Description' => t('This setting loads the page before executing Javascript which can improve your SEO.<br>
+                    <strong>**Warning: Enabling this feature may cause Javascript errors on your site.**</strong> ' .
+                    anchor(t('More information'), 'https://success.vanillaforums.com/entry/signin?Target=categories')),
```
Comment (slafleche) [inline applications/dashboard/controllers/class.settingscontroller.php]: Is this supposed to be hidden behind a login?

Comment (slafleche) [reply applications/dashboard/controllers/class.settingscontroller.php]: Also, can we get a `target="_blank"` on this link?

Comment (cchabilall83) [reply applications/dashboard/controllers/class.settingscontroller.php]: https://success.vanillaforums.com/kb/articles/add?draftID=606

File: applications/dashboard/controllers/class.settingscontroller.php [hunk 2]
```diff
@@ -557,6 +557,18 @@ public function branding() {
...
+            ],
+            'Feature.DeferredLegacyScripts.Enabled' => [
+                'LabelCode' => t('Defer Javascript Loading'),
+                'Control' => 'toggle',
+                'Default' => true,
+                'Description' => t('This setting loads the page before executing Javascript which can improve your SEO.<br>
+                    <strong>**Warning: Enabling this feature may cause Javascript errors on your site.**</strong> ' .
+                    anchor(t('More information'), 'https://success.vanillaforums.com/kb/articles/140-defer-javascript-loading-feature')),
+                'Options' => [
+                    'Multiline' => true,
```
Comment (charrondev) [inline applications/dashboard/controllers/class.settingscontroller.php]: I don't think this option is meant to be here.

```suggestion
```

**Pull request merged**